### ISSUE TITLE
Running transformers is stuck at a point

### ISSUE THREAD

**Issue by abhishekmg** [m87-labs/moondream#107]
I am trying to run the code in the example 

```
from transformers import AutoModelForCausalLM, AutoTokenizer
from PIL import Image

model_id = "vikhyatk/moondream2"
revision = "2024-05-20"
model = AutoModelForCausalLM.from_pretrained(
    model_id, trust_remote_code=True, revision=revision
)
tokenizer = AutoTokenizer.from_pretrained(model_id, revision=revision)

image = Image.open('<IMAGE_PATH>')
enc_image = model.encode_image(image)
print(model.answer_question(enc_image, "Describe this image.", tokenizer))
```

But for some reason at the `model.answer_question(enc_image, "Describe this image.", tokenizer)` the program is stuck and when I KeyboardInterrupt it I get this as the trace 

<img width="1041" alt="image" src="https://github.com/vikhyat/moondream/assets/34393560/65cff9fc-2890-4c10-b43f-a28be08ebb7d"> 

Any help is much appreciated, thanks

**Comment by omarnahdi** [comment]
@abhishekmg ohh... I think it's your first time running the model and that's why hugging face is downloading it 😁. Not any issue with the code just wait for a couple of minutes based on your internet speed. After downloading the model it will run locally

**Comment by abhishekmg** [comment]
Yes it is my first time running the model😅 sure I’ll wait till it downloads 😄
